js/packages/web/src/contexts/tokenList.tsx

```diff
@@ -36,27 +36,29 @@ export function SPLTokenListProvider({ children = null as any }) {
       ...process.env.NEXT_SPL_TOKEN_MINTS.split(",")
     ]: [WRAPPED_SOL_MINT]
 
+  const { tokens } = useConnectionConfig();
+
   useEffect(() => {
-    new TokenListProvider().resolve().then(setTokenList);
-  }, [setTokenList]);
+    setTokenList(new TokenListContainer(Array.from(tokens.values())))
+  }, [setTokenList, tokens]);
 
   const hasOtherTokens = !!process.env.NEXT_SPL_TOKEN_MINTS;
 
   // Added tokenList to know in which currency the auction is (SOL or other SPL)
-  const mainnetTokens = tokenList?tokenList.filterByClusterSlug("mainnet-beta").getList().filter(f=> subscribedTokenMints.some(s=> s == f.address) )
+  const subscribedTokens = tokenList?tokenList.getList().filter(f=> subscribedTokenMints.some(s=> s == f.address) )
```

Comment:
Removing the hard-coded `"mainnet-beta"` from here is the whole purpose of this PR. Now that the `tokenList` is already filtered using whatever cluster is chosen by the user, this can be removed.